enforcer/remote/enforcerLauncher/interfaces.go

```diff
@@ -0,0 +1,11 @@
+package enforcerLauncher
```

Comment:
Name of this package does not fit with the go convention. More informations here https://blog.golang.org/package-names

Same for the name of the file